sdcm/cluster.py

```diff
@@ -1498,6 +1641,7 @@ def start_scylla(node, queue):
         self.log.debug('Update DB binary duration -> %s s', int(time_elapsed))
 
     def _update_db_packages(self, new_scylla_bin, node_list):
+        # fixme: only support centos distro
```

Comment:
What does this #fixme mean?

Reply:
_update_db_packages() isn't the common usecase, I want to support other distros later.